README.md

```diff
@@ -38,6 +38,30 @@ Go deeper into a topic or scenario, oftentimes using building block APIs togethe
 | [Observability](./tutorials/observability) | Demonstrates Dapr tracing capabilities. Uses Zipkin as a tracing component. |
 | [Secret Store](./tutorials/secretstore) | Demonstrates the use of Dapr Secrets API to access secret stores. |
 
+## Development
+
+### Updating sdk versions
```

Comment:
Can you add the other languages here too? 

I see the makefile has more than just `go` & `python` supported. This is super helpful - thank you for adding this! Just want to ensure its documented too for future reference :)